Documentatie/EP-online-validaties.md

```diff
@@ -49,6 +50,8 @@ Onderstaande validaties worden allemaal en in willekeurige volgorde uitgevoerd.
 |  CheckBuildingIdentifiedByBagHasUniqueBagIds  								|	Alle pand-id's (BagBuildingId) mogen slecht één keer voorkomen in het bestand.
 |  CheckBuildingObservation  													|	Opname heeft plaatsgevonden in het gebouw (BuildingObservation = Yes).
 |  CheckBuildingType  															|	Bij woningbouw: bij elk gebouw moet het gebouwtype (BuildingCategory) ingevuld zijn. <br/> Bij utiliteitsbouw: bij een gebouw mag zowel het gebouwtype (BuildingCategory) als het subtype (BuildingCategorySupplement) niet ingevuld zijn.
+|  CheckBuildingTypeBagBerth													|	Bij gebruik van BagBerthId zijn enkel gebouwtype (BuildingCategory) 13 (Woonboot bestaande ligplaats) en 15 (Woonboot nieuwe ligplaats) toegestaan.
+|  CheckBuildingTypeBagPitch													|	Bij gebruik van BagPitchId is enkel gebouwtype 14 (Woonwagen) toegestaan.
```

Comment:
Achter gebouwtype tussen haakjes (BuildingCategory) toevoegen, zoals bij andere checks ook is gedaan.